docs/ref/models/querysets.txt

```diff
@@ -651,6 +651,12 @@ rather than one-tuples. An example should make the difference clearer::
 
 It is an error to pass in ``flat`` when there is more than one field.
 
+You can pass ``named=True`` to get results as a
+:func:`~python:collections.namedtuple`::
+
+    >>> Entry.objects.values_list('id', 'headline', named=True)
+    <QuerySet [Row(id=1, headline='First entry'), ...]>
+
```

Comment:
To explain why `namedtuple` isn't the default (it seems backwards compatible as Anssi noted on the ticket), maybe we should add something like:
Using a named tuple may make use of the results more readable, at the expense
of a small performance penalty for transforming the results into a named tuple.

Reply:
+1